plainform.py

```diff
@@ -225,12 +225,11 @@ class DateField(DateField, Field):
     pass
 
 
-class DateTimeField(DateTimeField, Field):
-    pass
-
-
 class DateTimeLocalField(DateTimeLocalField, Field):
-    pass
+    def __init__(self, label=None, validators=None,
+                 format='%Y-%m-%dT%H:%M:%S', **kwargs):
```

Comment:
We have to repeat the whole signature to allow args to be given with or without explicit names.